src/styles/main.scss

```diff
@@ -1,3 +1,68 @@
+@mixin cell-propertis ($cell-width, $cell-height, $bg-color) {
+  width: $cell-width;
+  height: $cell-height;
+  background-color: $bg-color;
+}
+
+$space-size: 1px;
```

Comment:
this variable is used for the border width of the cell and for the gap between cells
It is not a great idea, as most probably these values should not be coupled.